src/components/DayPickerKeyboardShortcuts.jsx

```diff
@@ -176,33 +179,36 @@ class DayPickerKeyboardShortcuts extends React.PureComponent {
 
     return (
       <div>
-        <button
-          ref={this.setShowKeyboardShortcutsButtonRef}
-          {...css(
-            styles.DayPickerKeyboardShortcuts_buttonReset,
-            styles.DayPickerKeyboardShortcuts_show,
-            bottomRight && styles.DayPickerKeyboardShortcuts_show__bottomRight,
-            topRight && styles.DayPickerKeyboardShortcuts_show__topRight,
-            topLeft && styles.DayPickerKeyboardShortcuts_show__topLeft,
-          )}
-          type="button"
-          aria-label={toggleButtonText}
-          onClick={this.onShowKeyboardShortcutsButtonClick}
-          onMouseUp={(e) => {
-            e.currentTarget.blur();
-          }}
-        >
-          <span
-            {...css(
-              styles.DayPickerKeyboardShortcuts_showSpan,
-              bottomRight && styles.DayPickerKeyboardShortcuts_showSpan__bottomRight,
-              topRight && styles.DayPickerKeyboardShortcuts_showSpan__topRight,
-              topLeft && styles.DayPickerKeyboardShortcuts_showSpan__topLeft,
-            )}
-          >
+        { renderKeyboardShortcutsButton
+          ? renderKeyboardShortcutsButton() : (
+            <button
+              ref={this.setShowKeyboardShortcutsButtonRef}
+              {...css(
+                styles.DayPickerKeyboardShortcuts_buttonReset,
+                styles.DayPickerKeyboardShortcuts_show,
+                bottomRight && styles.DayPickerKeyboardShortcuts_show__bottomRight,
+                topRight && styles.DayPickerKeyboardShortcuts_show__topRight,
+                topLeft && styles.DayPickerKeyboardShortcuts_show__topLeft,
+              )}
+              type="button"
+              aria-label={toggleButtonText}
```

Comment:
I might have to change this to `this.getToggleButtonText()` and bind to this scope since it relies on state change in `DayPicker.jsx`

Reply:
I'm confused, what is being referred to as "custom button" here? Seems like `aria-label` would work here unless I misunderstand?

Reply:
@monokrome `toggleButtonText` here relies on `showKeyboardShortcuts` which is a state variable set in the parent `DayPicker.jsx`. At first, I didn't think there was a way for me to pass in a similar variable through `renderKeyboardsShortcutsButton` that can read the state of `DayPicker.jsx` unless I bound it somehow.

But actually though, I just realized that I can pass in `toggleButtonText` as a param into the `renderKeyboardShortcutsButton()` here and drill that back into the render logic. Turns out I don't need to change any logic here. Hope that made sense!

Reply:
Cool! Thanks so much for explaining to me 😺